nagios-plugins-bdii.spec

```diff
@@ -1,18 +1,14 @@
 Summary: Nagios Plugin - check_bdii_entries
 Name: nagios-plugins-bdii
-Version: 1.0.15
+Version: 1.0.16
 Release: 1%{?dist}
-# The source for this package was pulled from upstream's vcs.  Use the
-# following commands to generate the tarball:
-#   svn export http://svnweb.cern.ch/guest/gridinfo/nagios-plugins-bdii/tags/R_1_0_15 %{name}-%{version}
-#  tar --gzip -czvf %{name}-%{version}.tar.gz %{name}-%{version}
 Source0: %{name}-%{version}.tar.gz
 License: ASL 2.0
 Group: Applications/System
 BuildRoot: %{_tmppath}/%{name}-%{version}-build
 BuildRequires: openldap-devel%{?_isa}
```

Comment:
I would suggest to add here some other packages needed for build: make rsync gcc. I did that for other packages and used yum-utils to install the dependencies instead of listing them in travis.

See https://github.com/EGI-Foundation/nagios-plugin-notebooks/blob/master/.travis.yml#L14

Reply:
Done, thanks!